flowmachine/flowmachine/features/utilities/event_table_subset.py

```diff
@@ -164,7 +173,47 @@ def _check_dates(self):
                 stacklevel=2,
             )
 
-    def _make_query(self):
+    def _make_query_with_sqlalchemy(self):
+        table = get_sqlalchemy_table_definition(
+            self.table.fully_qualified_table_name, engine=Query.connection.engine
+        )
+        sqlalchemy_columns = [
```

Comment:
In the same vein as with `self.subscriber_subset_ORIG`, should we not also be initializing `self.column` and `self.table` directly as `sqlalchemy` objects in `__init__` and removing those calls here.

Reply:
Yes, that's a good suggestion. At first I wanted to keep the changes in this PR as confined as possible (to allow for gradual refactoring) but this seems to make more sense.